manifests/profile/dns/aws.pp

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2018 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::profile::dns::standard
+#
+# Set up standard resolv_conf.
+#
+# @example
+#   include nebula::profile::dns::aws
+class nebula::profile::dns::aws {
+
+  $searchpaths = lookup('nebula::resolv_conf::searchpath')
+  $searchpath_str = join($searchpaths, ' ')
+
+  exec { 'restart_networking':
+    command     => '/bin/systemctl restart networking',
+    refreshonly => true,
+  }
+
+  file_line {
```

Comment:
This probably seems obvious to those who do DNS/DHCP more often, but maybe a comment either here or at the class level would be good to say what we're expecting to get from hiera and the effect of this addition or replacement.

As with most comments, it's a balance -- I don't think we have to explain everything done by the tools we're configuring with puppet, but this change seems to encode some rationale/practice of ours I don't totally understand by just looking at this code. I think probably half of that is not knowing dhclient well and half of it is that we're doing something with .lib vs .umdl and bare hostnames.

Reply:
Added a class level comment when fixing the aforementioned copy-pasta. Clearer?

Reply:
Yup, looks great.